PocMarten/PocMarten.Api/Aggregates/Invoices/Events/InvoiceDateStarted.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore.Routing.Constraints;
+using PocMarten.Api.Common.EventSourcing;
+
+namespace PocMarten.Api.Aggregates.Invoices.Events
+{
+    public class InvoiceDateStarted : IEventState
+    {
+        public double Amount { get; set; }
```

Comment:
It's not criticall, but decimal should be used for money if Amount is something related to money.

Reply:
It's related. I will have it in mind.